src/toast/atm.py

```diff
@@ -17,13 +17,19 @@
 
 from .mpi import MPI, MPIShared
 
-from ._libtoast import (
-    atm_sim_compute_slice,
-    atm_sim_observe,
-    atm_sim_compress_flag_hits_rank,
-    atm_sim_compress_flag_extend_rank,
-    atm_sim_kolmogorov_init_rank,
-)
+available_atm = None
+if available_atm is None:
```

Comment:
This test is unnecessary

Reply:
What is your recommended way to ensure that the following import try/except is executed exactly once on first import of the file, and that the `available_atm` package variable is set to True or False for all subsequent uses?

Reply:
Hah! I didn't realize that the declaration of `available_atm` would not be executed more than once. Withdrawn.

Reply:
And back on the table. Seems like none of the code in a previously imported file will execute again

Reply:
I changed this and another place in the code with a similar pattern.